tests/test_packets.py

```diff
@@ -12,12 +12,21 @@
 from pymine_net import load_packet_map
 
 
-@pytest.mark.parametrize( # GameState: (clientbound, serverbound)
-    "protocol, config", (
-        [757, {GameState.HANDSHAKING: (None, 0x0), GameState.STATUS: (0x1, 0x1), GameState.LOGIN: (0x4, 0x2), GameState.PLAY: (0x67, 0x2F)}],
-    )
+@pytest.mark.parametrize(  # GameState: (clientbound, serverbound)
```

Comment:
Why is this reformatted, the black workflow should've taken care of this?

Reply:
ah, because you set up the black workflow to not affect the test dir. ill fix that in this as well if you want